Source/SpatialGDK/Private/SpatialNetDriver.cpp

```diff
@@ -737,7 +737,7 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 	// Not calling Super:: on purpose.
 	UNetDriver::TickDispatch(DeltaTime);
 
-	if (SpatialOSInstance != nullptr && SpatialOSInstance->GetEntityPipeline() != nullptr)
+	if (SpatialOSInstance != nullptr && SpatialOSInstance->GetEntityPipeline() != nullptr && SpatialOSInstance->GetEntityPipeline()->IsInitialized())
```

Comment:
Do you think we could put the `IsInitialized` in the `SpatialOSInstance` instead since it owns the entity pipeline? It'll reduce the if to being just one boolean and instead of two.

Reply:
In our case, yes, but in theory it doesn't have to be owned by it or initialized at the same time. This keeps the responsibility where it really is / in case anyone's crazy enough to write their own implementation at some point.

Doesn't matter anyway since it should go away soon?